src/sxastarter/src/components/LayoutContainer.tsx

```diff
@@ -23,6 +26,21 @@ export function parseInlineStyles(styles: string | undefined): Record<string, st
   return styleObject;
 }
 
+export function parseBreakpointsClass(size: number): string {
+  const breakpoints: Record<BreakpointKeys, number> = {
```

Comment:
There should be no hardcodes for breakpoints values

Reply:
Updated!